lldb/tools/yaml2macho-core/Utility.cpp

```diff
@@ -0,0 +1,22 @@
+//===-- Utility.cpp -------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "Utility.h"
+#include "CoreSpec.h"
+
+void add_uint64(const CoreSpec &spec, std::vector<uint8_t> &buf, uint64_t val) {
```

Comment:
This takes a `CoreSpec` but doesn't actually use it? Same for `add_uint32` below.

Reply:
Originally you could specify the Endianness in the YAML and I needed to pass that to the add_uint methods to fix byte ordering in the output file.  I eventually lost confidence that I was actually creating correctly-formatted Big Endian mach-o files and I removed that feature, forgot to clean up and remove this argument.